tensorflow/tools/dockerfiles/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Copyright 2018 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Should we rename this dockerfile to something like "assembler.Dockerfile" to help avoid the confusion with the current Dockerfile?

Reply:
Good idea, done.